experimental/adaptive-tool/README.md

```diff
@@ -12,32 +12,79 @@
 - press `F1` then type `LG live test` to start LG webview test
 
 ## Features
-### Debugging adaptive dialogs
-- Supporting adaptive dialog breaking point
-
-### Debugging language generation
-- Support LG template breaking points
-- Support Expression breaking points
+### Debugging
+[Setting up](#set_up)
+- supporting adaptive dialog breaking point
```

Comment:
Re-word to 

- Supports break points in .dialog files (adaptive dialog)
- Supports break points in .lg files
- Supports break points in adaptive expressions